app/views/items/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1><%= @item.name %></h1>
+
+<p>Description: <%= @item.description %></p>
+<p>Price: <%= number_to_currency(@item.unit_price / 100.00) %></p>
```

Comment:
Nice job converting to dollars and cents.  I wonder if we can make a helper method to do that conversion for us, so we don't have to think about it.  And that would allow us to test that it is returning what we want as well in a model test.